src/components/Lesson.vue

```diff
@@ -136,7 +137,7 @@ import Validator from './Validator.vue'
 import TutorialCompletionCallout from './callouts/TutorialCompletion.vue'
 import TypeIcon from './TypeIcon.vue'
 
-const MAX_EXEC_TIMEOUT = 10000
+const MAX_EXEC_TIMEOUT = isProduction ? 10e3 : 60e3
```

Comment:
@zebateira Can you teach me why these aren't numbers? Or is this some fancy number formatting I haven't learned yet?

Reply:
Figured it out but would suggest that these numbers are small enough that it makes sense to write them out to make the code easier for newcomers to follow.